traveler-ios-ui/TravelerKitUI/ViewControllers/BookableProductDetailViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  BookableProductDetailViewController.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-05-30.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+class BookableProductDetailViewController: UIViewController {
+    @IBOutlet weak var headerView: PagedHorizontalView!
+    @IBOutlet weak var productNameLabel: UILabel!
+    @IBOutlet weak var passDetailsHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var passDetailsView: UIView!
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var productDescriptionLabel: UILabel!
+    @IBOutlet weak var productInfoHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var productInfoView: UIView!
+
+    internal var purchasedProduct:BookableProduct?
+    internal var productDetails:CatalogItemDetails?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationController?.navigationBar.isTranslucent = true
```

Comment:
always white space line after super call

Reply:
This will indefinitely set the navigation bar as translucent, Id say remove the translucency for now